internal/bundles/manifest.go

```diff
@@ -171,6 +176,16 @@ func NewManifestFromConfig(cfg *config.Config) *Manifest {
 	}
 	if cfg.R != nil {
 		m.Platform = cfg.R.Version
+		// If the configuration specifies a specific R version constraint
+		// (e.g. ">=3.8"), declare the environment requires that version.
+		if cfg.R.RequiresRVersion != "" {
+			if m.Environment == nil {
+				m.Environment = &Environment{}
+			}
+			m.Environment.R = &EnvironmentR{
+				Requires: cfg.R.RequiresRVersion,
+			}
+		}
```

Comment:
It'd be nice to have a `NewManifestFromConfig` unit test that includes `Environment` being populated with this info

Reply:
Added in `TestNewManifestFromConfigVersionRequirements`